src/Raven.Server/ServerWide/ServerStore.cs

```diff
@@ -3029,20 +3014,21 @@ public DatabaseTopology LoadDatabaseTopology(string databaseName)
             }
         }
 
-        public async Task<(long Index, object Result)> SendToLeaderAsync(CommandBase cmd)
+        // TODO: We need always to link the token with the request abort token
```

Comment:
TODO?

Reply:
if we accept this PR I'll create a ticket to handle that, didn't want to make changes which aren't directly related to the issue here.